<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWindow</class>
 <widget class="QMainWindow" name="CalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatorWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>20</width>
      <height>721</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_journal_list">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Журнал</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action_result">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>=</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_inputexpression">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action_plus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>+</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action_minus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>-</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>3</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>0</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action_comma">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>,</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action_point">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>.</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>2</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action_multiply">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>*</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>6</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>5</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>4</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>7</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>8</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_number_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>9</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action_divide">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>/</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выражение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Постфиксная запись выражения</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_postfix_expression">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_input">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>функции</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_action_sin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>sin()</string>
      </property>
      <property name="text">
       <string>sin()</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_action_cos">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>cos()</string>
      </property>
      <property name="text">
       <string>cos()</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_action_log">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>log()</string>
      </property>
      <property name="text">
       <string>log()</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_action_ln">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>ln()</string>
      </property>
      <property name="text">
       <string>ln()</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_action_e">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>e</string>
      </property>
      <property name="text">
       <string>e</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_action_Pi">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Pi</string>
      </property>
      <property name="text">
       <string>Pi</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Скобки</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_action_close_square_bracket">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>]</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_action_open_round_bracket">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_action_close_round_bracket">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_action_open_square_bracket">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_action_degree">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>^</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_journal">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>321</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action_balance">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>О программе</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
